cypress/support/commands.js

```diff
@@ -83,15 +82,21 @@ Cypress.Commands.add('registerUser', () => {
       failOnStatusCode: false,
     }).then((response) => {
       if (response.status === 200) {
-        cy.request(
-          'PATCH',
-          `${apiServer}/submit-verification-code`,
-          {
-            email: response.body.email,
-            twoFactorToken: response.body.twoFactorToken,
-          },
-        ).then((response2) => {
-          expect(response2.status).to.eq(200);
+        cy.waitForLatestEmail('cb52ca14-0d64-4821-9970-4e81b47e13c1').then((mail) => {
+          assert.isDefined(mail);
+          token = /token=([A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*)/.exec(mail.body)[1];
```

Comment:
This var could be initialized in the same line
```suggestion
          const token = /token=([A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*)/.exec(mail.body)[1];
```

Reply:
Done.